ionic2/config.xml

```diff
@@ -1,61 +1,62 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
No it has always been used to store installed cordova plugins and options (like the icons or the splash screen).